.changeset/purple-analysts-depart.md

```diff
@@ -0,0 +1,43 @@
+---
+"@fluidframework/tree": minor
+---
+
+Support generation of JSON Schema from Shared Tree view schema (alpha)
+
+Adds alpha-quality support for canonical [JSON Schema](https://json-schema.org/docs) representation of Shared Tree schema and adds a `getJsonSchema` function for getting that representation for a given `TreeNodeSchema`.
+This `JSON Schema` representation can be used to describe schema requirements to external systems, and can be used with validation tools like [ajv](https://ajv.js.org/) to validate data before inserting it into a Shared Tree.
+
+Example:
```

Comment:
```suggestion
### Example
```

Reply:
What is the "root" heading level of a changeset? I considered doing this, but it wasn't clear to me what the right level was (if any). Is this documented somewhere?

Reply:
(Updated to use your recommendation, but I would still love to see the heading level expectations for this documented somewhere if they aren't already).

Reply:
There's only every supposed to be one H1, so the headings in the release notes start at H2. But we can't expect everyone to always get this right, so we will continue to have reviews and whatnot for release notes. And with the previews, it'll be obvious if someone gets it wrong.

Reply:
Then why was the recommendation to be a third-level heading? Should it be second-level? Or was the suggested level mostly arbitrary? If we're going to recommend devs use headings in their changelogs, I'd like for them to be consistent.